modules/web-mercator/src/fit-bounds.d.ts

```diff
@@ -26,18 +26,18 @@ type Padding = {
  *  properties defining the padding.
  * @param [options.offset] - The center of the given bounds relative to the map's center,
  */
-type FitBoundsOptions = {
+export type FitBoundsOptions = {
```

Comment:
Types are more useful to your TS users if they are exported.

Reply:
Good point. 

I suppose it is a balance as it is also good to be careful with the public API - any change to these would be a breaking changes. If they are public, we would want to think through naming conventions etc before publishing them, just like we do with all exported JS symbols.